cmd/main.go

```diff
@@ -113,7 +117,7 @@ func main() {
 	buildReconciler := &controller.OpenShiftBuildReconciler{
 		Client:     mgr.GetClient(),
 		Scheme:     mgr.GetScheme(),
-		Shipwright: shipwrightbuild.New(mgr.GetClient()),
+		Shipwright: shipwrightbuild.New(mgr.GetClient(), common.OpenShiftBuilNamespaceName),
```

Comment:
I'd simply pass this in here, or do something like

```suggestion
                namespace := common.FetchCurrentNamespace()
                setupLog.Info("operator namespace found", "namespace", namespace)
		Shipwright: shipwrightbuild.New(mgr.GetClient(),  namespace),
```